### PR TITLE
Push of week 10/17

### PR DESCRIPTION
**Pull request by huntress284**
Remake of branch, creation of pug files

### CODE REVIEW

File: judgingCriteria.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title> YCP Hacks </title>
+    <!-- CSS only -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet"
```
Comment (davidmchugh2000) [inline judgingCriteria.html:8]: Test comment

**Review by dile0 — APPROVED** [review]

File: simple/views/welcomePage.pug [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+            br
+            |                 
+            h3 One
+            |                 
+            p
+              .container.mt-3
+                table.table.table-striped
+                  tbody
+                    tr
+                      td 4:00-5:00
```
Comment (davidmchugh2000) [inline simple/views/welcomePage.pug:81]: How simple will this be to abstract later? How easy will it be for a user to change?

File: simple/index.mjs [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+import express from 'express';
+import mysqlx from '@mysql/xdevapi';
+
+import config from './config.mjs';
+
+const app = express();
+
+app.set('view engine', 'pug');
+
+const port = process.env.PORT || 7777;
```
Comment (davidmchugh2000) [inline simple/index.mjs:12]: This port should probably be held in a constants file

Comment (tfranks31) [reply simple/index.mjs:12]: I agree, magic numbers are typically frowned upon

**Review by tfranks31 — COMMENTED** [review]
Good start, make sure to implement and structure properly as you go along. I know there is a desire to show clients visual progress during standup, but it isn't worth it if you have to go back and change your code to implement it "properly".  Don't play all your cards at once, and don't create loads of technical debt for yourselves, RT has suffered from it in the past.

Also, make sure to remove `testFile.txt`

**Comment by davidmchugh2000** [comment]
Please push before Monday @tfranks31 approves

**Review by tfranks31 — APPROVED** [review]

**Pull request merged**